src/com/vinnik/kate/MultithreadingSorter.java

```diff
@@ -0,0 +1,56 @@
+package com.vinnik.kate;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+/** Class, which creates given number of threads, and measures time they need to sort random array. */
+public class MultithreadingSorter {
+
+    /** Creates given number of threads and executes them. */
+    public static void measureTimeOfSorting(int amountOfThreads) {
+        for (int i = 0; i < amountOfThreads; i++) {
+            ThreadSort thread = new ThreadSort(i);
+            thread.start();
+        }
+    }
+
+
+    private static class ThreadSort extends Thread {
+        private int[] numbersToSort;
+        private int key;
+
+        protected ThreadSort(int number) {
+            numbersToSort = new int[500];
+            key = number;
+
+            for (int i = 0; i < numbersToSort.length; i++) {
+                Random random = new Random();
+                numbersToSort[i] = random.nextInt(10000);
```

Comment:
`10000` is a magic number. Consider put it into class's constants saying that it's an upper bound for a random value which could be generated.